Makefile

```diff
@@ -133,7 +134,7 @@ uninstall:
 
 clean:
 	@rm -v -f $(DISTTARGET) vim-src/.config.h-modified vim-src/src/*.orig
-	@cd vim-src && $(MAKE) -s distclean && hg update -C
+	@cd vim-src && $(MAKE) -s distclean && ( git reset --hard; git clean -x -f -d -q; )
```

Comment:
- There's no reason to use `make -s distclean` because the following Git commands will blow everything away.
- I generally avoid letting my lines extend beyond printed 79 characters. Please separate the lines. For Makefiles, I treat tabs as 8-space tabs. 
- There's no reason to use a subshell, i.e. "(...)" here instead of a command group, i.e. "{...}". You can read about the differences here: https://www.gnu.org/software/bash/manual/html_node/Command-Grouping.html#Command-Grouping